analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpSyntaxHelper.cs

```diff
@@ -282,8 +284,7 @@ node switch
         public static bool HasConstantValue(this ExpressionSyntax expression, SemanticModel semanticModel) =>
             expression.RemoveParentheses().IsAnyKind(LiteralSyntaxKinds) || expression.FindConstantValue(semanticModel) != null;
 
-        public static string GetStringValue(this SyntaxNode node, SemanticModel semanticModel) =>
-            CsharpStringValueHelper.Instance.GetStringValue(node, semanticModel);
+        public static string GetStringValue(this SyntaxNode node, SemanticModel semanticModel) => CSharpFacade.Instance.Syntax.NodeStringTextValue(node, semanticModel);
```

Comment:
This is an indirection to my eyes - am I missing something?
If yes - I'll open an optimization issue.